src/checkers/inference/model/serialization/TestSerialization.java

```diff
@@ -49,8 +48,8 @@ public void testEquality() {
         VariableSlot slot2 = InferenceMain.getInstance().getSlotManager().createVariableSlot(AnnotationLocation.MISSING_LOCATION);
 
         // Test that order does not matter
-        EqualityConstraint eqConst1 = constraintManager.createEqualityConstraint(slot1, slot2);
-        EqualityConstraint eqConst1a = constraintManager.createEqualityConstraint(slot2, slot1);
+        Constraint eqConst1 = constraintManager.createEqualityConstraint(slot1, slot2);
```

Comment:
What if I wan't to assert `slot1` and `slot2` are not changed before calling and after calling `constraintManager.createEqualityConstraint(slot1, slot2)`?

Reply:
I don't understand the source of the problem here. Such a test is not even possible without additional constant slots. Why do have this random test requirement now?